liquidator/liquidator.js

```diff
@@ -118,70 +112,82 @@ class Liquidator {
     // expired and disputed liquidations.
     const expiredLiquidations = this.empClient.getExpiredLiquidations();
     const disputedLiquidations = this.empClient.getDisputedLiquidations();
-    const potentialWithdrawableLiquidations = expiredLiquidations.concat(disputedLiquidations);
+    const potentialWithdrawableLiquidations = expiredLiquidations
+      .concat(disputedLiquidations)
+      .filter(liquidation => liquidation.liquidation === this.account);
```

Comment:
+1 this is a good change. no point in submitting these tx's unless they originated from the `this.account`.